api/python/cellxgene_census/src/cellxgene_census/_open.py

```diff
@@ -179,7 +202,13 @@ def get_source_h5ad_uri(dataset_id: str, *, census_version: str = DEFAULT_CENSUS
         's3_region': 'us-west-2'}
     """
     description = get_census_version_description(census_version)  # raises
-    census = _open_soma(description["soma"])
+
+    # For h5ads, it makes sense to use the default mirror, since the artifacts themselves won't be mirrored
```

Comment:
I don't think this assumption is true in the longer term.  We have the same motivation to mirror all large datasets